.changelog/1978.internal.md

```diff
@@ -0,0 +1 @@
+Add endSlot prop in AlertBox component
```

Comment:
`internal` is used for builds, docs, refactor etc. Simple tasks that are reflected in UI are using `trivial` type.